.travis.yml

```diff
@@ -15,14 +15,25 @@ jdk:
 env:
   global:
   - GRADLE_OPTS="-Xms128m"
-
+  - secure: nt4Ta5rI0asqHT4Y3B9JpMYaKdrdMbVVFyN9VZ7vAk1N3jmc/a2sEioIKqhVLtnrI/4GDfbR9hd6y27fnC+4ZsUFkxpglDH3XISqqJyt28saDS4OlpVldPEhV16QIHwANC+AAUiT99dQeNSfmmVnTuJAVLxmBILmmJdG0Rkt+73isZJCLIldtJu+UPNK9OgWXz7nIJV03b5Rw+JsU+PEtiduOZav0hXQvhJ9ojDaIhy2Ac8sKYC4UKTQmsLCQYT4YPgp9zvRnrzAtOQFgnlRKdgudEcLrRymjMwW5BXBn/iuT7MTbyfrWkXgAlG07d/RJ+TU5bh4bo5mYxa+e1dU3lmcr8yYsEYc0lljz9BWfkd7NpmwtbcQf2TcaEoUOBJ/HeaVtrWVrPjLwe8qVXzvnYRfYU6fSIU27Pskswzsf/qi+2OEnO42FTcmuiFEDvJDDrleEHFKZk5XNr/MelAmfJTKx1IUx8aNbga29oqBmvYQMnBqM4BlJ2zv2YBnDUdqk6pBRPR4EXOxxx8XPhePQ8HprypNnxGBUAj2qSIkdyp84PX1GTP5PCw43kbzKfFfG2Bd669EXB2piFPS9f9rPESJ4kCoR8nkfg16R1SJvuGDH984nlwGgGPc0Ku7UzIhsxkeTwEniAKd12zHDTzicz/mUWznok1ntV/ZEyZL5bk=
```

Comment:
we don't really need the variables declared twice, either declare it globally or as part of the snapshot task.